R/expr.R

```diff
@@ -245,9 +248,36 @@ expr_label <- function(expr) {
     paste0("`", chr, "`")
   }
 }
-
+#' @rdname expr_label
 #' @export
+expr_name <- function(expr) {
+  name <- switch_type(expr,
+    symbol = as_name(expr),
+    quosure = ,
+    language = {
+      expr_text <- deparse(expr)
+      paste0("(", expr_text, ")")
```

Comment:
This needs the same logic as `expr_label`, i.e. insert `...` if the call is too long